src/strategy.py

```diff
@@ -32,9 +32,12 @@ def __init__(self, constants, start_time, **kwargs):
         self.interval = kwargs.get('interval')  # required
         self.current_time = start_time - self.interval
         # relative allowed difference between battery SoC and desired SoC when leaving
-        self.margin = 0.05
+        self.margin = 0.1
```

Comment:
is this change intended?

Reply:
Yes, the simulations showed that it appears quite often, that the desired_soc is not fully reached. Therefore, I think a little bit more default tolerance ist useful.

Reply:
This has nothing to do with this PR, so a separate commit would have been cleaner. However, as it is only a minor change, it can be included if you think it's beneficial. We still have our strategy check that the vehicles can really make their trip, even when leaving with less than their desired SoC.